README.md

```diff
@@ -1,23 +1,25 @@
 
 ## Getting Started
-This document assumes that you are aware about Github and if you are coming from svn or other versioning system it is recommended to follow the link https://github.com/Seagate/cortx/blob/main/doc/github-process-readme.md to configure git on your local machine. Following Readme document will give you enough insights and start contributing.
+This document assumes that you are aware about Github and if you are coming from SVN or other code versioning system it is recommended to follow the link [Github Process Readme](https://github.com/Seagate/cortx/blob/main/doc/github-process-readme.md) to configure git on your local machine. Following Readme document will give you enough insights to start contributing.
 
-You can have a separate client VM with any Linux flavour to install client side pre-requisites and start using automation framework on the same VM. This VM should have connectivity to Cortx Cluster OR CORTX OVA deployment. Alternatively you may use one of the nodes as client (less recommended).     
+You need a separate client VM with any Linux flavour (prefer CentOS 7+ ) to install client side pre-requisites and start running automation framework on the same VM. This VM should have network connectivity to a Cortx Cluster OR CORTX OVA deployment. Alternatively you may use one of the nodes as client (less recommended).     
 
 ## Git process
-Typically a member contributing to test framework would follow the review process as follows:
-	1. We are following the concept of upstream and downstream where commits happen on your forked repository (downstream)
-	2. Then you can raise a PR to merge it to Seagate Cortx-Test repository (Upstream)
-	3. Moderators of Cortx-Test can create server side feature branch if multiple developers are working on same feature branch
-	4. Team member should be able to check-in and raise the PR to upstream even if they have read-only access to Seagate Repositories using this process
+Typically, a member contributing to test framework would follow the review process as follows:
+1. Commits happen on your forked repository(origin).  
+2. Then you can raise a PR to merge it to Seagate Cortx-Test repository (Upstream)
+3. Moderators of Cortx-Test can create server side feature branch if multiple developers are working on same feature branch
+4. Team member should be able to check-in and raise the PR to upstream even if they have read-only access to Seagate Repositories using this process
```

Comment:
Is point no. 4 an extension of the point no. 2? If yes, maybe we can re-word the 2nd point as follows -

2. Then you can raise a PR to merge it to Seagate Cortx-Test repository (Upstream). PRs can be raised even if you have read-only access to Seagate Repositories.

Reply:
I am OK with this rewording or point#4 can be added as a footnote as well instead of separate bullet. Not a major change, good to have.

Reply:
anyway the repo is public now.